### ISSUE TITLE
Concise documentation needed

### ISSUE THREAD

**Issue by emahuni** [syntacticsolutions/Laravel-Vue-Quasar#1]
I think this is a great starter, but it needs a bit more flesh in the day on how to go for those like me who're trying to merge the two.

**Comment by syntacticsolutions** [comment]
Yeah sorry. I have to tweak it a little bit in order to get it right and I
haven't had time to mess with it. This was more of an experiment, but if
you have any questions I would be glad to help. You could also contribute
to the documentation if you so choose.
